src/harmony/DatePicker/DatePicker.module.css

```diff
@@ -0,0 +1,98 @@
+.DatePicker {}
+
+.DatePickerOption {
+    display: flex;
+    gap: var(--gap-s);
+
+    margin-bottom: var(--gap-sm);
+}
+
+.DatePickerOptionLabel {
+    width: 94px;
+}
+
+.DatePickerOptionContent {
+    width: 252px;
+}
+
+/* Chrome, Safari, Edge, Opera */
+.DatePicker input::-webkit-outer-spin-button,
+.DatePicker input::-webkit-inner-spin-button,
+.DatePicker input::-webkit-calendar-picker-indicator {
+  -webkit-appearance: none;
+  appearance: none;
+  margin: 0;
+  user-select: none;
+  pointer-events: none;
+  height: 0;
+  width: 0;
+  display: none;
+}
+
+.DatePicker input[type="date"] {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    font-family: inherit;
+}
+
+/* Firefox */
+.DatePicker input[type=number] {
+    -moz-appearance: textfield;
+}
+
+.DatePickerFooter {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.DatePickerResetControl {
+    font-size: var(--font-size-xs);
+    cursor: pointer;
+    border-bottom: 1px dotted currentColor;
+}
+
+.DatePickerInlineWrapper {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+}
+
+.DatePickerInlineGap {
+    gap: var(--gap-xs);
+}
+
+.DatePickerQuarter {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.DatePickerQuarterAlias {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+}
+
+.DatePickerStrictDateInput {
+    width: 2ch;
+    text-align: center;
+}
+
+.DatePickerYearInput {
+    text-align: center;
+}
+
+.DatePickerYearInput, .DatePickerStrictDateInput[name=year] {
+    width: 4ch;
+}
+
+.DatePickerInlineWrapper + .DatePickerInlineWrapper,
+.DatePickerQuarter + .DatePickerQuarterAlias {
+    margin-top: var(--gap-xs);
+}
+
+.DatePickerStrictDateAdvice {
+    padding-left: calc(95px + var(--gap-s));
```

Comment:
95px?

Reply:
This offset for correct alignment by right column

Reply:
Nobody knows. Use comments to explain magic.